src/app/app.routes.ts

```diff
@@ -0,0 +1,3 @@
+import { Routes } from '@angular/router';
+
+export const routes: Routes = [];
```

Comment:
suggestion

          export const routes: Routes = []; // Define an empty routes array

          Explantion: Adding a comment helps others understand the purpose of this empty routes array which is intended for route definitions.